build-system/common/check-package-manager.js

```diff
@@ -63,34 +63,36 @@ function yellow(text) {
 }
 
 /**
- * @fileoverview Perform checks on the AMP toolchain.
- */
-
-// If npm is being run, print a message and cause 'npm install' to fail.
-function ensureYarn() {
-  if (process.env.npm_execpath.indexOf('yarn') === -1) {
+ * If yarn is being run, print a message and cause 'yarn install' to fail.
+ **/
+function ensureNpm() {
+  if (process.env.npm_execpath.indexOf('npm') === -1) {
```

Comment:
```suggestion
  if (process.env.npm_execpath.includes('npm')) {
```
Also nit: Maybe add a comment with a link to the documentation/StackOverflow thread discussing this env var